games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
Can you point out please where the issue with sci-libs/libspatialindex lies? I cant find any reference to it and i dont even have it installed on my system and yet everything seems to compile just fine.